lib/css/stacks.css

```diff
@@ -8154,6 +8154,12 @@ body {
   font-size: 13px;
   line-height: 1.30769231;
 }
+@media (max-width: 640px) {
+  html ,
+  html  body {
```

Comment:
This whitespace is a little weird, no?

Reply:
Yeah, admittedly it bothered me a little bit as well 😁 But it's not like it's an actual issue, especially in generated code that's going to be minified anyway.